js/rpg_managers/DataManager.js

```diff
@@ -41,6 +41,8 @@ DataManager._globalId       = 'RPGMV';
 DataManager._lastAccessedId = 1;
 DataManager._errorUrl       = null;
 
+DataManager.imageReservationId  = Utils.generateRuntimeId();
```

Comment:
I see, this is implemented before implementing default id mechanism.
move to Scene_File looks good.